### PR TITLE
Add branding part of CMS

### PR DESCRIPTION
**Pull request by joehoyle**
This removes from WordPress bits, adds a color scheme for Platform and option to set a logo for login

### CODE REVIEW

**Comment by joehoyle** [comment]
Depends on #1

File: inc/branding/namespace.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace HM\Platform\CMS\Branding;
+
+function bootstrap() {
```
Comment (rmccue) [inline inc/branding/namespace.php]: Needs phpDoc

File: inc/branding/namespace.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+/**
+ * Remove dashboard widgets that are not useful.
+ */
+function remove_dashboard_widgets() {
+	remove_meta_box( 'dashboard_primary', [ 'dashboard', 'dashboard-network', 'dashboard-user' ], 'side' );
+}
+
+function add_color_scheme() {
```
Comment (rmccue) [inline inc/branding/namespace.php]: Needs phpDoc

File: inc/branding/namespace.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+			'#152354',
+			'#14568A',
+			'#D54E21',
+			'#2683AE',
+		],
+		[
+			'base' => '#152354',
+			'focus' => '#fff',
+			'current' => '#fff',
+		]
```
Comment (rmccue) [inline inc/branding/namespace.php]: Could we move these into a constant or something instead?

File: inc/namespace.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace HM\Platform\CMS;
+
+use function HM\Platform\get_config;
+
+function bootstrap() {
```
Comment (rmccue) [inline inc/namespace.php]: Needs phpDoc

File: inc/namespace.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	require_once __DIR__ . '/remove_updates/namespace.php';
+	require_once __DIR__ . '/branding/namespace.php';
```
Comment (rmccue) [inline inc/namespace.php]: Can we move these to `load.php` instead? No need to conditionally load them in.

File: inc/namespace.php [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+	if ( $config['login-logo'] ) {
+		add_action( 'login_header', __NAMESPACE__ . '\\add_login_logo' );
+	}
+}
+
+/**
+ * Add the custom login logo to the login page.
+ *
+ * @return void
```
Comment (rmccue) [inline inc/namespace.php]: I wouldn't bother with this, but if you do, be consistent and use it on all the other functions too.

File: inc/remove_updates/namespace.php [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace HM\Platform\CMS\Remove_Updates;
+
+function bootstrap() {
```
Comment (rmccue) [inline inc/remove_updates/namespace.php]: Needs phpDoc

File: inc/remove_updates/namespace.php [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+	add_action( 'admin_init', 'HM\\Platform\\CMS\\Remove_Updates\\remove_update_nag' );
+	add_action( 'plugins_loaded', 'HM\\Platform\\CMS\\Remove_Updates\\remove_update_check_cron' );
```
Comment (rmccue) [inline inc/remove_updates/namespace.php]: `__NAMESPACE__` here instead.

File: inc/branding/namespace.php (hunk 3 above)
Comment (joehoyle) [reply inc/branding/namespace.php]: nah

Comment (joehoyle) [reply inc/branding/namespace.php]: these are just for the svgs, so I don't think it's worth it really

File: assets/admin-color-scheme.css [hunk 9]
```diff
@@ -0,0 +1,504 @@
...
+ */
+ body {
+	background: #f1f1f1;
+  }
+
+  /* Links */
+  a {
+  color: #0073aa;
+  awd: #4667de;
+  awd: #152a4e;
```
[the remark this answers is not in the document]
Comment (joehoyle) [reply assets/admin-color-scheme.css]: my bad! ....move along

**Pull request merged**